docs/Operators.md

```diff
@@ -1161,13 +1159,13 @@ Other versions of this operator: <a href="Changelog.md#BatchNormalization-1">Bat
 <dt><tt>Y</tt> : T</dt>
 <dd>The output tensor of the same shape as X.</dd>
 <dt><tt>mean</tt> (optional) : T</dt>
-<dd>The running mean after the BatchNormalization operator. Must be in-place with the input mean. Should not be used for testing.</dd>
+<dd>The running mean after the BatchNormalization operator. Must be in-place with the input mean.</dd>
```

Comment:
Omit "Must be in-place with the input mean"

Reply:
will fix.